README.md

```diff
@@ -54,6 +54,13 @@ make test # all tests
 make test TEST="-k product" # single module test
 ```
 
+The tests are run against prerecorded server-responses.
+If you need to (re-)record some cassettes, delete the stale ones in `test/test_playbooks/fixtures` and record new ones with
+
+```sh
+pytest -k <module name> --record
```

Comment:
Yeah, or `make test TEST="-k <module name> --record"`

Reply:
sounds a bit more bulky, and if you are at that stage (recording a new tape) you might have run the test setup already. But for consistency...
We could make it `make record TEST=...`.

Reply:
Nah, let's leave it as is.